content/en/docs/concepts/scheduling-eviction/eviction-policy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Eviction Policy
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+This page is an overview of Kubernetes' policy for eviction.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Eviction Policy
```

Comment:
Note that eviction can be triggered by other components: the scheduler during preemption, also I think node lifecycle controller evicts pods when NoExecute taints are violated.

Reply:
_for now_ I'm happy with a simple copy-and-paste. My thinking was to then log another issue to capture the need for more subtle tidying.